axelrod/tests/strategies/test_memory_decay.py

```diff
@@ -0,0 +1,78 @@
+"""Tests for the Memory Decay strategy"""
```

Comment:
If the strategy is going to stay in the `meta.py` module, have the tests in the `test_meta.py` module please.